src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -7816,7 +7816,7 @@ export class StudyViewPageStore
         if (this.shouldDisplayClinicalEventTypeCounts.result) {
             this.changeChartVisibility(
                 SpecialChartsUniqueKeyEnum.CLINICAL_EVENT_TYPE_COUNTS,
-                true
+                false
             );
```

Comment:
@ShahKirtan2003 wondering if this whole block can be removed? I guess the default is to hide the chart?

Reply:
Yes @inodb, removing the changeChartVisibility block is the same as setting `false` value. Addressed your suggestion in latest commit.